From be370fe0143aa618f3f6a39442c421031208c359 Mon Sep 17 00:00:00 2001
From: Clark Wang <xiaoning.wang@nxp.com>
Date: Mon, 19 Nov 2018 10:30:48 +0800
Subject: [PATCH 5194/5242] MLK-20351-2 Doc: fsl-imx-cspi: Add a description
 of dmas cells

commit  aa6f3c56c10573c60ec42a45df65cc4f19e0878c from
https://source.codeaurora.org/external/imx/linux-imx.git

The description of dmas pointed to the common description of dma,
and the introduction of dmas was unclear. Add a doc link to sdma
introduction.

Signed-off-by: Clark Wang <xiaoning.wang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../devicetree/bindings/spi/fsl-imx-cspi.txt       |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/spi/fsl-imx-cspi.txt b/Documentation/devicetree/bindings/spi/fsl-imx-cspi.txt
index e3c48b2..8555d90 100644
--- a/Documentation/devicetree/bindings/spi/fsl-imx-cspi.txt
+++ b/Documentation/devicetree/bindings/spi/fsl-imx-cspi.txt
@@ -27,6 +27,8 @@ Optional properties:
 - num-cs :  Number of total chip selects, see spi-bus.txt.
 - dmas: DMA specifiers for tx and rx dma. See the DMA client binding,
 Documentation/devicetree/bindings/dma/dma.txt.
+	For the description of the second cell of dma phandle, see this document,
+		Documentation/devicetree/bindings/dma/fsl-imx-sdma.txt
 - dma-names: DMA request names, if present, should include "tx" and "rx".
 - fsl,spi-rdy-drctl: Integer, representing the value of DRCTL, the register
 controlling the SPI_READY handling. Note that to enable the DRCTL consideration,
-- 
1.7.9.5

